packages/botonic-core/src/router.js

```diff
@@ -205,6 +205,7 @@ export class Router {
       if (input.type == 'text') value = input.data
     } else if (prop == 'input') value = input
     else if (prop == 'session') value = session
+    else if (prop == 'request') value = { input, session }
```

Comment:
If we do the changes explained in another comment, should we add here the `lastRoutePath`?